### PR TITLE
EB KB API Boxes

### PR DESCRIPTION
**Pull request by brandysmith**

### CODE REVIEW

File: ElasticBox/api-boxes.md [hunk 1]
```diff
@@ -0,0 +1,1771 @@
...
+
+![box2.png](../images/ElasticBox/box2.png)
+
+**Response Parameters**
+
+![box3.png](../images/ElasticBox/box3.png)  ![box4.png](../images/ElasticBox/box4.png)
+
+```
+[
+        {
```
Comment (srveit) [inline ElasticBox/api-boxes.md]: Remove extra spaces from beginning of line. Leave just two spaces.

File: ElasticBox/api-boxes.md [hunk 2]
```diff
@@ -0,0 +1,1771 @@
...
+    "name": "Redis",
+    "id": "7e846aa1-d490-4b8a-a2a1-555407a90105",
+    "members": [],
+    "owner": "public",
+    "organization": "public",
+    "events": {},
+    "draft_from": "aa5a019a-5dd6-4669-a591-ec52783b123e",
+    "icon": "images/platform/redis.png"
+  },
+{
```
Comment (srveit) [inline ElasticBox/api-boxes.md]: Indent two spaces

File: ElasticBox/api-boxes.md [hunk 3]
```diff
@@ -0,0 +1,1771 @@
...
+
+        },
+        {
+            "required":true,
+            "type":"Port",
+            "name":"Port_variable",
+            "value":"80",
+            "visibility":"public"
+
+        },{
```
Comment (srveit) [inline ElasticBox/api-boxes.md]: Add newline after comma.

File: ElasticBox/api-boxes.md [hunk 4]
```diff
@@ -0,0 +1,1771 @@
...
+
+        },
+        {
+            "name":"Binding_db",
+            "type":"Binding",
+            "value":"2a7a5f6b-280d-47de-afaa-65db5dd85816",
+            "visibility":"public"
+
+        }
+        ],
```
Comment (srveit) [inline ElasticBox/api-boxes.md]: Change indent to 4 spaces for this line and the following 8 lines.

**Comment by rpdubose** [comment]
@srveit Why can we not have the spaces? This is just a copy & paste from eb.com. That doesn't make it right but it is rendering correctly over there. Do we have css that we automatically do the indent?

**Comment by srveit** [comment]
@rpdubose The JSON is inside of a code block, so the spacing will be exactly as written. It just looks more professional if the indentation is consistent.

**Comment by brandysmith** [comment]
Fixe code block spacing

File: ElasticBox/api-boxes.md [hunk 5]
```diff
@@ -0,0 +1,1772 @@
...
+    "name": "Redis",
+    "id": "7e846aa1-d490-4b8a-a2a1-555407a90105",
+    "members": [],
+    "owner": "public",
+    "organization": "public",
+    "events": {},
+    "draft_from": "aa5a019a-5dd6-4669-a591-ec52783b123e",
+    "icon": "images/platform/redis.png"
+    },
+{
```
Comment (jim-phillips) [inline ElasticBox/api-boxes.md]: indent to match preview `}`

File: ElasticBox/api-boxes.md [hunk 6]
```diff
@@ -0,0 +1,1772 @@
...
+            "type":"Box"
+
+        },
+        {
+            "name":"Binding_db",
+            "type":"Binding",
+            "value":"2a7a5f6b-280d-47de-afaa-65db5dd85816",
+            "visibility":"public"
+
+    }
```
Comment (jim-phillips) [inline ElasticBox/api-boxes.md]: indent to match previous `{`

File: ElasticBox/api-boxes.md [hunk 7]
```diff
@@ -0,0 +1,1772 @@
...
+         "Type":"String",
+         "MinLength":"1",
+         "MaxLength":"255",
+         "AllowedPattern":"[\\x20-\\x7E]*",
+         "ConstraintDescription":"can contain only ASCII characters."
+      },
+    ...
+   }
+...
+...
```
Comment (jim-phillips) [inline ElasticBox/api-boxes.md]: are 2 `...` necessary?

File: ElasticBox/api-boxes.md [hunk 8]
```diff
@@ -0,0 +1,1772 @@
...
+         "Type":"String",
+         "MinLength":"1",
+         "MaxLength":"255",
+         "AllowedPattern":"[\\x20-\\x7E]*",
+         "ConstraintDescription":"can contain only ASCII characters."
+      },
+    ...
+   },
+...
+...
```
Comment (jim-phillips) [inline ElasticBox/api-boxes.md]: double `...`

**Comment by brandysmith** [comment]
Fixed code block spacing

**Pull request merged**